### PR TITLE
TM-33 - In the Quinovas cloudfront module, hard-code the region for the default-index-redirect lambda function.

### PR DESCRIPTION
**Pull request by rob-hargraves**

### CODE REVIEW

**Comment by rob-hargraves** [comment]
This has been tested and works in the new prodb environment.

File: aws/cloudfront/default-index-redirect.tf [hunk 1]
```diff
@@ -13,6 +13,11 @@ data "aws_iam_policy_document" "redirector_assume_role" {
   }
 }
 
+provider "aws" {
+  alias  = "lambda_edge_region"
+  region = "us-east-1"
+    }
```
Comment (mdillavou) [inline aws/cloudfront/default-index-redirect.tf]: Can you quickly fix the formatting here.

Comment (rob-hargraves) [reply aws/cloudfront/default-index-redirect.tf]: Oops, done.

**Comment by mdillavou** [comment]
Merged in master in d000bb869b21d1290844209253c20a4f701cd97f

**Pull request merged**